[
{
"cve_id": "CVE-2024-28114",
"patch_date": [
"2024-01-25 19:04:36+01:00"
],
"patch_files": [
"docs/templating/index.md",
"peering_manager/jinja2/__init__.py"
],
"earliest_patch_date": "2024-01-25 19:04:36+01:00",
"earliest_vuln_date": "2023-08-07 00:13:04+02:00",
"vuln_files": [
"peering_manager/jinja2/__init__.py"
],
"impact": {},
"cwes": [],
"fix_duplicate_type": false,
"fix_duplicate_types": [],
"intro_duplicate_type": false,
"intro_duplicated_types": [],
"file_fixed_before_fixed_again": false,
"files_fixed_before_fixed_again": [],
"file_fixed_before_vuln_again": false,
"files_fixed_before_vuln_again": []
},
{
"cve_id": "CVE-2024-28113",
"patch_date": [
"2024-01-26 01:12:59+01:00"
],
"patch_files": [
"utils/views.py"
],
"earliest_patch_date": "2024-01-26 01:12:59+01:00",
"earliest_vuln_date": "2023-07-19 22:06:05+02:00",
"vuln_files": [
"utils/views.py"
],
"impact": {},
"cwes": [],
"fix_duplicate_type": false,
"fix_duplicate_types": [],
"intro_duplicate_type": false,
"intro_duplicated_types": [],
"file_fixed_before_fixed_again": false,
"files_fixed_before_fixed_again": [],
"file_fixed_before_vuln_again": false,
"files_fixed_before_vuln_again": []
}
]